lib/api-get.js

```diff
@@ -41,11 +41,23 @@ class ApiGet extends API {
 				});
 		}
 
-		const response = this.format ? await this.format(record[0]) : record[0];
+		[this.record] = record;
+
+		await this.postGetValidate(this.record);
```

Comment:
Seguiría con la misma idea del `format()` y haría..
```suggestion
if(this.postGetValidate)
  await this.postGetValidate(this.record);
```
De esta forma no hacemos el pasamanos sin sentido